fe/src/main/java/org/apache/doris/alter/SchemaChangeJobV2.java

```diff
@@ -240,7 +240,8 @@ protected void runPendingJob() throws AlterCancelException {
                                     Partition.PARTITION_INIT_VERSION, Partition.PARTITION_INIT_VERSION_HASH,
                                     tbl.getKeysType(), TStorageType.COLUMN, storageMedium,
                                     shadowSchema, bfColumns, bfFpp, countDownLatch, indexes,
-                                    tbl.isInMemory());
+                                    tbl.isInMemory(),
+                                    tbl.getPartitionInfo().getTabletType(partitionId));
                             createReplicaTask.setBaseTablet(partitionIndexTabletMap.get(partitionId, shadowIdxId).get(shadowTabletId), originSchemaHash);
```

Comment:
You should add constraint about ALTER TABLE.
If I have a disk table.
```
ALTER TABLE test SET("tablet_type"="memory");
```
will be encounter error in BE processing.